Documentation/gettingstarted/kind.rst

```diff
@@ -53,6 +53,12 @@ Then, install Cilium release via Helm:
       --set image.pullPolicy=IfNotPresent \\
       --set ipam.mode=kubernetes
 
+.. note::
+
+   To fully enable Cilium's kube-proxy replacement (:ref:`kubeproxy-free`), cgroup v1
+   controllers ``net_cls`` and ``net_prio`` have to be disabled, or cgroup v1 has
```

Comment:
Should this indicate how to disable the controllers?

Reply:
Not sure whether it's worth, as Docker or other involved component might not work due to a missing v1 controller.